README.md

```diff
@@ -23,31 +23,49 @@ with the json files in the configs folder of rbac-config.
 Contributing
 =============
 
-Add new roles
+Canned roles
 -------------
+
+Add new roles
+
 Follow existing examples to add roles including name, description, system flag, access with permissions.
 If you want the new role to be associated with platform default group (which defines the default permissions
 for principals in a tenant), you have to add the platform_default flag and set it as true.
-
 Set the version to 2 for the new role in order to trigger the seeding in the rbac service.
-
-Format of permissions
-~~~~~~~~~~~~~~~~~~~~~
-The permission should have the format of <application>:<resource>:<action>. There are no restrictions on the
+If you have a new permission added, don't forget to add it to the permissions folder too. See "Add new permissions"
+section below for more details.
+~~~~~~~~~~~~~~~~~~~~~~
+Format of permissions:
+The permission should have the format of `<application>:<resource>:<action>`. There are no restrictions on the
 resource or action word, i.e. you could define any resource or action. It is up to the app team to determine how to use the
 resource and action themselves.
+~~~~~~~~~~~~~~~~~~~~~~
 
 Update roles
--------------
+
 When you update the role, please change the version number of the role for the service to pick up new features.
 If you update the name of the role, it will generate a new role but keep the old one. Please reach out to 
 RBAC team to resolve this.
 
 Delete roles
--------------
+
 Currently, if you delete a role from the config, it will still exist. Please reach out to RBAC team to delete them from database.
 
 
+Add new permissions
```

Comment:
I think it might be good to have a README.md in each of the directories for /permissions/ and /configs/ with a sample of what they should look like?

Reply:
Good idea, I will add a README.md in those directories. Only samples in the README, right?

Reply:
Turns out the create configmap will use the README file, therefore it could not be added inside the directories. But I added a line to let them refer to existing files for samples.